web/server/vue-cli/src/components/Report/ReportFilter/Filters/DetectionStatusFilter.vue

```diff
@@ -14,6 +14,37 @@
     </template>
 
     <template v-slot:append-toolbar-title>
+      <tooltip-help-icon>
+        Filter reports by the <b>latest</b> detection status.<br><br>
+
+        The detection status is the latest state of a bug report in a run. When
+        storing the results of a run from scratch then each report has
```

Comment:
Maybe it would be better to rephrase this part to something when a report id is first detected it will be marked as new not only when the reports are stored from scratch.